src/main/java/aquality/selenium/core/utilities/JsonSettingsFile.java

```diff
@@ -55,6 +57,8 @@ private Object castEnvOrDefaulValue(JsonNode node, String envVar) {
             return envVar == null ? node.asInt() : Integer.parseInt(envVar);
         } else if (node.isDouble()) {
             return envVar == null ? node.asDouble() : Double.parseDouble(envVar);
+        } else if (node.isObject()) {
```

Comment:
sonar doesn't like the complexity of this method

Reply:
I know.